client/cli/src/actions/build.ts

```diff
@@ -7,9 +8,15 @@ export default (config: DcmConfig) =>
   registerCommand({
     name: 'build',
     description: 'Build dynamic component',
-    options: [['-m, --mode <mode>', 'mode', STRING, BuildMode.development], ['-f --file <file>', 'file', STRING]],
-    action: ({ options: { mode } }) => {
-      let buildConfig: BuildOptions = { mode, modifyRollupConfig: config && config.modifyRollupConfig };
-      return build(buildConfig);
+    options: [['-m, --mode <mode>', 'mode', STRING, BuildMode.development], ['-d, --dir <directory>', 'dir', STRING]],
+    action: async ({ options: { mode, dir }, logger }) => {
+      const dirs = getComponentDirectories(dir, config && config.workspaces);
+
+      for (const dir of dirs) {
```

Comment:
why not use `await Promise.all(dirs.map(handler))`?

Reply:
So that if it fails it won't continue. I can add a parallel option

Reply:
parallel is a great idea here